test/logster/middleware/test_viewer.rb

```diff
@@ -18,6 +18,7 @@ def setup
 
   def teardown
     Logster.config.subdirectory = nil
+    Logster.store.redis.flushall
```

Comment:
maybe we should avoid this atomic destruction and instead kill off the parts of the store we know we touched? that keeps our "clean up redis mess" routines honest and working.